### PR TITLE
add primary key necessity.

### PR DESCRIPTION
**Pull request by icefairy**
If your table has no primary key,you will not see any data in flink (no matter history or new ) and has no error message.

### CODE REVIEW

**Comment by icefairy** [comment]
cloud you add this hint in FAQ? I cannot found the way to edit the FAQ document.

**Comment by ruanhang1993** [comment]
cc @leonardBang

File: docs/content/connectors/mysql-cdc.md [hunk 1]
```diff
@@ -76,6 +76,8 @@ When an initial consistent snapshot is made for large databases, your establishe
 - `interactive_timeout`: The number of seconds the server waits for activity on an interactive connection before closing it. See [MySQL documentations](https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_interactive_timeout).
 - `wait_timeout`: The number of seconds the server waits for activity on a noninteractive connection before closing it. See [MySQL documentations](https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_wait_timeout).
 
+### Make sure your table in mysql has primary key
+If your table has no primary key,you will not see any data in flink (no matter history or new ) and has no error message.
```
Comment (leonardBang) [inline docs/content/connectors/mysql-cdc.md:80]: Thanks @icefairy for the contribution, Could you add this as a NOTE to incremental snapshot section as following?
```
**NOTE:** Incremental snapshot requires primary key now, if your table has no primary key, you will not see any data as well as error message.
```

**Comment by ruanhang1993** [comment]
The tables without primary keys have been supported in 2.4.0. So I close this PR now. Thanks.

**Pull request closed without merging**